docs/workflows/api-token.md

```diff
@@ -397,3 +431,35 @@ func main() {
 
 </TabItem>
 </Tabs>
+
+## Validating Access Tokens:
+
+There are two types of tokens that can be validated: `self-contained` or `referential`. When validating self-contained tokens, consider if you want to revoke your tokens. If revoked tokens are accepted, then the token can be validated [offline](api-token#offline-validation) by validating the signature and parsing the claims of the JWT token. In all other cases the token should be [introspected](api-token#introspection-online-validation).
+
+### Introspection (Online Validation)
+
+A successful introspect query will return information about the token. This endpoint is compliant with [RFC-7662](https://www.rfc-editor.org/rfc/rfc7662). After receiving information about the token, validate the following:
+
+1. Check that either the application id or resource server identifier is listed in the `aud` of the token. If at least one of the allowed audiences is in the token `aud` claim, then the token is granted.
+2. Check that token has the expected scopes.
+
+```bash title="/introspect"
+curl https://api-$REGION.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/introspect \
+-X POST \
+-H "Authorization: Bearer $TOKEN" \
```

Comment:
Let's change this call to `Authorization: Basic $CLIENT_ID:$CLIENT_SECRET`. The client credentials need to belong to the confidential application that is introspecting the token.